src/types/UiaTracing.cpp

```diff
@@ -15,6 +15,28 @@ TRACELOGGING_DEFINE_PROVIDER(g_UiaProviderTraceProvider,
 
 using namespace Microsoft::Console::Types;
 
+// The first valid ID is 1 for each of our traced UIA object types
+IdType UiaTracing::_utrId = 1;
+IdType UiaTracing::_siupId = 1;
+
+void UiaTracing::_assignId(UiaTextRangeBase& utr) noexcept

```

Comment:
This seems weird to me that this isn't being done by the respective classes anymore. Could you add doc comments to help explain why `UiaTracing` is responsible for tracking the IDs like this?